package.json

```diff
@@ -108,6 +113,8 @@
     "yalc": "^1.0.0-pre.53"
   },
   "peerDependencies": {
+    "i18n-iso-countries": "^7.14.0",
```

Comment:
do you know what was the initial idea for this package about peerDependencies and normal dependencies? I'm just curious, because this ones are now the first like smaller packages instead of just our react basic ones 🤔